config/checks/run-checks.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+rem Runs all check-*.bat scripts, and returns a non-zero exit code if any of them fail.
+
+setlocal enabledelayedexpansion
+set ret=0
+
+for /R %%X in (check-*.bat) do (
+ call %%X
```

Comment:
I think it might be better to place `%%X` in double quotes for repo locations that happen to have a space like mine. Seems to work after

Reply:
Good catch! I did not realize this as I do not have any spaces.

Reply:
hmm can't say im the most experienced with batch scripts but it seems the removal of the `/R` actually fixed it so maybe there's no need to add the quotation marks